packages/contracts/docs/osx/03-reference-guide/index.md

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
This file should not be deleted but moved to `docs/developer-portal`

Reply:
good catch. 
fixed with d5b3530ca42fa04fe50297fdad31015479adf0a3